src/renderers/dom/ReactDOM.js

```diff
@@ -35,6 +37,8 @@ var React = {
   render: render,
   unmountComponentAtNode: ReactMount.unmountComponentAtNode,
   version: ReactVersion,
+  addDevtool: ReactDOMDevtools.addDevtool,
+  removeDevtool: ReactDOMDevtools.removeDevtool,
```

Comment:
These should be on the `__REACT_DEVTOOLS_GLOBAL_HOOK__` somehow because there is no reliable way to get to a single instance of a renderer from a global debugger protocol.